### ISSUE TITLE
PIXI not defined (via npm)

### ISSUE THREAD

**Issue by mauromazzei** [bigtimebuddy/pixi-svg#10]
Hi ,
i've installed pixi-svg and imported as an es6 but during usage it throws an error
> SVG.js:12 Uncaught ReferenceError: PIXI is not defined

Seems like it needs PIXI saved in the window object
To avoid this error i need to use like this
```
import * as PIXI from 'pixi.js'
window.PIXI = PIXI
import {SVG} from 'pixi-svg'

const svg = new PIXI.SVG(document.getElementById('svg1').outerHTML);
```

Thank You!

**Comment by bigtimebuddy** [comment]
I swear that I have solved this in v2. Can you please provide more details (how are you bundling, which version of pixi & pixi-svg) or, ideally, a repo that reproduces the problem?

**Comment by mauromazzei** [comment]
i don't have a repo yet but i'm the early stage of a test so i don't have any code if not the one that i've attached you.

pixi version: 5.2.4
pixi-svg: 2.1.0
bundling with parcel: 1.12.4

and this is the svg i'm using to trace
```
<svg class="road" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
     <circle class="road__path" r="25" cx="30" cy="30" />
</svg>
```

**Comment by bigtimebuddy** [comment]
Okay, I figured it out, the issue is we are defining `browser` field.
https://parceljs.org/module_resolution.html#package.json-%60browser%60-field

I have fixed this and made a new release (2.1.1). Here's a working example:
https://github.com/bigtimebuddy/pixi-svg-parcel-example
